config/routes.rb

```diff
@@ -32,5 +32,7 @@
     resources :payment_methods, except: [:delete]
     resources :student_payment_methods, only: [:create, :update]
 
+    get :me, to: 'me#show'
```

Comment:
Lo mismo que le puse a María;
podés poner esto:
```suggestion
    resources :me, only: [:show]
```